Assets/HoloToolkit/Interaction/Scripts/Receivers/ToggleActiveReceiver.cs

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using UnityEngine;
+using System.Collections;
+using System;
+using HoloToolkit.Unity.InputModule;
+
+namespace HoloToolkit.Unity.Receivers
+{
+    /// <summary>
+    /// Simple receiver class for toggling a game object active or inactive.
+    /// </summary>
+    public class ToggleActiveReceiver : InteractionReceiver
+    {
+		// When selected toggle color change
+		protected override void InputDown(GameObject obj, InputEventData args)
+		{
+            Debug.Log("On tapped in interaction receiver");
```

Comment:
nit: formatting.

Reply:
What am I missing here on formatting? I've updated the code comments for Input down but the other formatting looks valid to me.

thanks,
-pat

Reply:
Could be tabs vs spaces

Reply:
Hmm.. they should all be set to spaces.  Let me double check.  I'm not seeing the usual discrepancies in the GitHub webview I usually do when it's tab/space mangled.  Will double check.

thanks,
-pat